### PR TITLE
feature/27828_report_list_tabs: Split up the report view according to scan type

### PR DESCRIPTION
**Pull request by magenta-af**

### CODE REVIEW

File: django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -45,6 +45,26 @@ msgstr ""
 "Listen af regler kan ikke redigeres. For at scanne med andre regler, opret "
 "da et nyt scannerjob."
 
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:3
+msgid "Alle rapporter"
+msgstr ""
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:5
+msgid "Web rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po]: When in doubt, use compound nouns:
```suggestion
msgid "Webrapporter"
```

File: django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -45,6 +45,26 @@ msgstr ""
...
+msgstr ""
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:8
+msgid "Fil rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po]:
```suggestion
msgid "Filrapporter"
```

File: django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -45,6 +45,26 @@ msgstr ""
...
+msgstr ""
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:11
+msgid "Exchange rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/da/LC_MESSAGES/django.po]:
```suggestion
msgid "Exchangerapporter"
```

File: django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -46,6 +46,26 @@ msgstr ""
 "The list of rules cannot be edited. To scan with other rules, create a new "
 "scanner job."
 
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:3
+msgid "Alle rapporter"
+msgstr "All reports"
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:5
+msgid "Web rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po]:
```suggestion
msgid "Webrapporter"
```

File: django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -46,6 +46,26 @@ msgstr ""
...
+msgstr "Web reports"
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:8
+msgid "Fil rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po]:
```suggestion
msgid "Filrapporter"
```

File: django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -46,6 +46,26 @@ msgstr ""
...
+msgstr "File reports"
+
+#: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html:11
+msgid "Exchange rapporter"
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/locale/en/LC_MESSAGES/django.po]:
```suggestion
msgid "Exchangerapporter"
```

File: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html [hunk 7]
```diff
@@ -1,6 +1,15 @@
 <p class="lead">Se rapporter for specifikke scannerjob eller få en overbliksrapport for flere scannerjob.</p>
   <ul class="nav nav-tabs" role="tablist">
-    <li role="presentation"{% if active == 'reports' %} class="active"{% endif %}><a href="/reports/" role="tab">Scannerjob</a></li>
-    <li role="presentation"{% if active == 'summaries' %} class="active"{% endif %}><a href="/reports/summaries/" role="tab">Overblik</a></li>
+    <li role="presentation"{% if active == 'all' %} class="active"{% endif %}><a href="/reports/" role="tab">{% trans "Alle rapporter" %}</a></li>
+    {% if settings.ENABLE_WEBSCAN %}
+    <li role="presentation"{% if active == 'web' %} class="active"{% endif %}><a href="/reports/web/" role="tab">{% trans "Web rapporter" %}</a></li>
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html]:
```suggestion
    <li role="presentation"{% if active == 'web' %} class="active"{% endif %}>
      <a href="/reports/web/" role="tab">{% trans "Webrapporter" %}</a>
    </li>
```

File: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html [hunk 8]
```diff
@@ -1,6 +1,15 @@
...
+    {% endif %}
+    {% if settings.ENABLE_FILESCAN %}
+    <li role="presentation"{% if active == 'file' %} class="active"{% endif %}><a href="/reports/file/" role="tab">{% trans "Fil rapporter" %}</a></li>
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html]:
```suggestion
    <li role="presentation"{% if active == 'file' %} class="active"{% endif %}>
      <a href="/reports/file/" role="tab">{% trans "Filrapporter" %}</a>
    </li>
```

File: django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html [hunk 9]
```diff
@@ -1,6 +1,15 @@
...
+    {% endif %}
+    {% if settings.ENABLE_EXCHANGESCAN %}
+    <li role="presentation"{% if active == 'exchange' %} class="active"{% endif %}><a href="/reports/exchange/" role="tab">{% trans "Exchange rapporter" %}</a></li>
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/templates/os2webscanner/summaries_above_content.html]:
```suggestion
    <li role="presentation"{% if active == 'exchange' %} class="active"{% endif %}>
      <a href="/reports/exchange/" role="tab">{% trans "Exchangerapporter" %}</a>
    </li>
```

File: django-os2webscanner/os2webscanner/templates/os2webscanner/reports.html [hunk 10]
```diff
@@ -8,14 +8,14 @@
 
 
 <div class="container">
-{% include 'os2webscanner/summaries_above_content.html' with active='reports' %}
+{% include 'os2webscanner/summaries_above_content.html' with active=active %}
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane active" id="reports">
     <p class="h2">Scannerjob</p>
```
Comment (danchr) [inline django-os2webscanner/os2webscanner/templates/os2webscanner/reports.html]: I think we should remove this header; it's largely redundant now.

**Comment by dals83** [comment]
small_ui_bug
Found this small bug in the interface.

**Pull request merged**